cms/static/js/views/pages/container.js

```diff
@@ -40,11 +40,19 @@ define(['jquery', 'underscore', 'backbone', 'gettext', 'js/views/pages/base_page
             initialize: function(options) {
                 BasePage.prototype.initialize.call(this, options);
                 this.viewClass = options.viewClass || this.defaultViewClass;
+                this.isUnitPage = this.options.isUnitPage;
+                this.isSplitTest = (this.model.attributes.category === 'split_test');
```

Comment:
If we always add the access editor (without these checks) we end up with the editor being added to the library page, is there a better way that we can do this check?